frontend/src/lib/actions/postClaim.js

```diff
@@ -0,0 +1,45 @@
+import { fail } from '@sveltejs/kit';
+import { invalidateAll } from '$app/navigation';
+import { applyAction, deserialize } from '$app/forms';
+
+/** @type {any} */
+export let form;
+
+const BASE_URL = 'http://0.0.0.0:5555';
```

Comment:
needs to read from the env var defined in the docker file, and default to the local host.